mkdocs.yml

```diff
@@ -1,124 +1,207 @@
-dev_addr: '127.0.0.1:8001'
-edit_uri: edit/main/nautobot/docs
-site_dir: nautobot/project-static/docs
-site_name: Nautobot Documentation
-site_url: https://nautobot.readthedocs.io/
-repo_url: https://github.com/nautobot/nautobot
+---
+dev_addr: "127.0.0.1:8001"
+edit_uri: "edit/main/nautobot/docs"
+site_dir: "nautobot/project-static/docs"
+site_name: "Nautobot Documentation"
+site_url: "https://nautobot.readthedocs.io/"
```

Comment:
To address #2345, this *maybe* should be `https://nautobot.readthedocs.io/en/latest/` or `https://nautobot.readthedocs.io/en/stable/`. I see that the 404 page rendering is broken on https://nbatest-ml.readthedocs.io/projects/core/en/latest/development/404, although it seems to be working on https://nautobot--2347.org.readthedocs.build/en/2347/development/404...

Reply:
Setting the `site_url` to the full `https://nautobot.readthedocs.io/en/stable/` does the trick. The CI RTD build for the PR is a fluke and the fact that it is working without the fix actually worries me that it's not representative of the production setup :(

For future reference - there is an [open issue on RTD](https://github.com/readthedocs/readthedocs.org/issues/4820) about injecting the correct canonical tag in mkdocs builds, but it seems to be stalled.